lib/mix/tasks/hexweb/check_names.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Mix.Tasks.Hexweb.CheckNames do
+  use Mix.Task
+  require Logger
+
+  @shortdoc "Check package names for typosquatters"
+
+  def run(_args) do
+    Mix.Task.run "app.start"
+
+    threshold = Application.get_env(:hex_web, :levenshtein_threshold)
+
+    threshold
+    |> find_candidates
+    |> send_mail(threshold)
+
+    :ok
+  end
+
+  def find_candidates(threshold) do
+    querystr = """
+      SELECT pnew.name new_name, pall.name curr_name, levenshtein(pall.name, pnew.name) as dist
+      FROM packages as pall
+      CROSS JOIN packages as pnew
+      WHERE pall.name <> pnew.name
+        AND pnew.inserted_at >= CURRENT_DATE
+        AND levenshtein(pall.name, pnew.name) <= $1
+        ORDER BY pall.name, dist;
+    """
```

Comment:
You don't need to indent the content of the heredoc. It's common in Elixir to indent the content on the same level as the final `"""`, i.e.:

```elixir
foo = """
bar
baz
bong
"""
```

:)